week2/materials/homework/assignments.md

```diff
@@ -4,27 +4,28 @@
 
 Sync your changes to s3 and write down your s3 bucket url.
 
-bucket url: `?`
+bucket url: `https://s3.console.aws.amazon.com/s3/buckets/soujanya-week2-homework?region=us-east-1&tab=objects`
 
 **Assignment 5:**
 
 Right now, the website does not support https. Explain which other AWS service needs to be integrated in order to achieve this.
 
-service name: `?`
+service name: `AWS Certificate Manager`
```

Comment:
Yes would be nice to link to some resource here for later documentation to read up but correct

Reply:
@bytewiz  Ok, :) 👍